xapian-applications/omega/index_file.h

```diff
@@ -126,6 +126,16 @@ void
 index_add_document(const std::string & urlterm, time_t last_altered,
 		   Xapian::docid did, const Xapian::Document & doc);
 
+void
+index_ctime_terms(Xapian::Document & doc, DirectoryIterator & d,
+		  const time_t & mtime, const time_t & ctime);
```

Comment:
This new function is only used in `index_file.cc`.  Considering what it does, I think it makes more sense to treat it as an internal implementation detail (so add `static` to the definition in the `.cc`, and don't declare it here in the header).

Reply:
Okay, It makes sense. I will do it.